fabric.ts

```diff
@@ -1,425 +1,57 @@
-import { Polyline } from './src/shapes/Polyline';
-import { Polygon } from './src/shapes/Polygon';
-import { Text } from './src/shapes/Text/Text';
-import { IText } from './src/shapes/IText/IText';
-import { Textbox } from './src/shapes/Textbox';
-import { Group } from './src/shapes/Group';
-import { ActiveSelection } from './src/shapes/ActiveSelection';
-import { Image } from './src/shapes/Image';
-import { getEnv, getDocument, getWindow, setEnvForTests } from './src/env';
-import { createCollectionMixin } from './src/Collection';
-import { parseAttributes } from './src/parser/parseAttributes';
-import { parseElements } from './src/parser/parseElements';
-import { getFilterBackend } from './src/filters/FilterBackend';
-import { parseStyleAttribute } from './src/parser/parseStyleAttribute';
-import { parsePointsAttribute } from './src/parser/parsePointsAttribute';
-import { parseTransformAttribute } from './src/parser/parseTransformAttribute';
-import { getCSSRules } from './src/parser/getCSSRules';
-
-import './src/controls/default_controls';
+export { getEnv, getDocument, getWindow, setEnvForTests } from './src/env';
+export { cache } from './src/cache';
+export { VERSION as version, iMatrix } from './src/constants';
+export { config } from './src/config';
+export { runningAnimations } from './src/util/animation/AnimationRegistry';
+
+export { Observable } from './src/Observable';
+
+export { StaticCanvas } from './src/canvas/StaticCanvas';
+export { Canvas } from './src/canvas/Canvas';
+
+export { Point } from './src/Point';
+export { Intersection } from './src/Intersection';
+export { Color } from './src/color/Color';
+
+export { Gradient } from './src/gradient/Gradient';
+export { Pattern } from './src/Pattern';
+export { Shadow } from './src/Shadow';
+
+export { BaseBrush } from './src/brushes/BaseBrush';
+export { PencilBrush } from './src/brushes/PencilBrush';
+export { CircleBrush } from './src/brushes/CircleBrush';
+export { SprayBrush } from './src/brushes/SprayBrush';
+export { PatternBrush } from './src/brushes/PatternBrush';
+
+export { FabricObject as Object } from './src/shapes/Object/FabricObject';
+export { Line } from './src/shapes/Line';
+export { Circle } from './src/shapes/Circle';
+export { Triangle } from './src/shapes/Triangle';
+export { Ellipse } from './src/shapes/Ellipse';
+export { Rect } from './src/shapes/Rect';
+export { Path } from './src/shapes/Path';
+export { Polyline } from './src/shapes/Polyline';
+export { Polygon } from './src/shapes/Polygon';
+export { Text } from './src/shapes/Text/Text';
+export { IText } from './src/shapes/IText/IText';
+export { Textbox } from './src/shapes/Textbox';
+export { Group } from './src/shapes/Group';
+export { ActiveSelection } from './src/shapes/ActiveSelection';
+export { Image } from './src/shapes/Image';
+export { createCollectionMixin } from './src/Collection';
+
+export * as util from './src/util';
+
+export * from './src/parser';
```

Comment:
Not being a named export this seems just hides visibility. Can we see the named exports here?

Reply:
I don't mind but is missing the whole point of moving it to a standalone index
I can return them here

Reply:
I want them to be
`export * as parser from './src/parser';`